EIPS/eip-draft_location-metadata-standard.md

```diff
@@ -0,0 +1,189 @@
+
+Moreover, inconsistent data formats present a rather challenging situation if an user wants to make use of this data, as it can be computationally expensive and slow to clean it up. This standard provides a solution by setting a standard which will cover the need of having location metadata and also give the community a framework which will prove tremendously useful at the application layer.
+
+One example where consistent location data structures are crucial is in supply chain management. As business partners try to coordinate their inventory to eliminate the potential for stockouts or spoilage, they need to understand the location of tokenized inputs. Tracking these inputs on one interface is only possible if all tokens are using the same standards for recording location. 
+
+## Specification
+
+This EIP extends the implemenation of the Asset Metadata JSON in the ERC-721 tokens; as such, the only technical requirement is to write the JSON file as follows:
+
+```json
+{
+  "title": "Asset Metadata",
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Identifies the asset to which this NFT represents"
+    },
+    "description": {
+      "type": "string",
+      "description": "Describes the asset to which this NFT represents"
+    },
+    "image": {
+      "type": "string",
+      "description": "A URI pointing to a resource with mime type image/* representing the asset to which this NFT represents. Consider making any images at a width between 320 and 1080 pixels and aspect ratio between 1.91:1 and 4:5 inclusive."
+    },
+    "location": {
+      "type": "object",
+      "name": {
+        "type": "string",
+        "description": "Identifies the place where the asset represented by this NFT is located."
+      },
+      "geolocation": {
+        "type": "object",
+        "latitude": {
+          "type": "string",
+          "description": "Estimates the geographic position of the asset to which this NFT represents. This value is a representation of the latitude using ISO 6709."
+        },
+        "longitude": {
+          "type": "string",
+          "description": "Estimates the geographic position of the asset to which this NFT represents. This value is a representation of the longitude using ISO 6709."
+        },
+        "altitude": {
+          "type": "string",
+          "description": "Estimates the geographic position of the asset to which this NFT represents. This value is a representation of the altitude using ISO 6709."
+        },
+        "crsid": {
+          "type": "string",
+          "description": "Identifies the coordinate reference system (CRS or SRID) being used. Default is WGS84."
+        }
```

Comment:
possibility of adding a field that lists supported mapping apps (google maps, open street maps, GPS devices)

Reply:
This is not contemplated within the scope of the metadata itself but the EIP itself can present use cases and apps/platforms where the standard would be supported once they do.